open_schools_platform/organization_management/circles/migrations/0004_reverse_location.py

```diff
@@ -0,0 +1,21 @@
+from django.db import migrations
+
+
+def reverse_circles_location(apps, schema_editor):
+    circles = apps.get_model('circles', 'Circle')
+    for circle in circles.objects.all():
+        x = circle.location.x
+        circle.location.x = circle.location.y
+        circle.location.y = x
+        circle.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('circles', '0003_circle_location'),
+    ]
+
+    operations = [
+        migrations.RunPython(reverse_circles_location)
```

Comment:
also make return migrations method as second argument in RunPython
like that: migrations.RunPython(create_profiles_for_employees, delete_employees_profiles),